### PR TITLE
No implicit any

### LINKED ISSUE #1: Missing method return types
There are multiple places where a method return type is missing. This cause the Typescript compiler to fail when the noImplicitAny option is on. Some examples:

```
IRouterProps {
   ...
   stringifyQuery?(queryObject: Object);
   onError?(error: Error);
}
```

### PR DESCRIPTION
**Pull request by vslinko**
Fixes #1

### CODE REVIEW

**Comment by vslinko** [comment]
@andrew-w-ross ping

**Comment by andrew-w-ross** [comment]
@vslinko  Hey sorry was away, I'll have a look at this later.

**Comment by vslinko** [comment]
@andrew-w-ross you can see diff without whitespace changes:
https://github.com/vslinko/typings-react-router/commit/87a7fd7a3fe8f3026ecc3eb01f19d2046d531c42?w=1

File: react-router.d.ts [hunk 1]
```diff
@@ -110,70 +110,70 @@ declare module ReactRouter {
...
 
 		/**
 		 * The styles to apply to the link element when its route is active.
-		 * 
-		 * @param {React.MouseEvent} event **NOTE** I'm assuming this will be a click event. 
+		 *
+		 * @param {React.MouseEvent} event **NOTE** I'm assuming this will be a click event.
 		 */
-		onClick?(event: React.MouseEvent);
+		onClick?(event: React.MouseEvent): any;
```
Comment (vslinko) [inline react-router.d.ts:168]: I think we could use `React.MouseEventHandler` from `react` definitions:

``` diff
-       onClick?(event: React.MouseEvent): any;
+       onClick?: React.MouseEventHandler;
```

**Comment by teddybradford** [comment]
@andrew-w-ross Have you had a chance to look at this PR? I'd love to see this merged. 👍

**Comment by andrew-w-ross** [comment]
@vslinko @teddybradford @palmsey @malekpour 

Hey I'm extremely sorry this has taken so long for me to get to in general.
Would any of you like access to this and my other typings repositories?

I'm going to be doing some backend code for the next two months so my concentration won't be here for the meantime.

**Comment by vslinko** [comment]
@andrew-w-ross I could be a maintainer of those packages.

**Comment by andrew-w-ross** [comment]
@vslinko You're added, If I missed out any just give me a shout.

**Comment by malekpour** [comment]
Not sure how much time I can spend, but you can add me as backup if you
would like.

**Comment by palmsey** [comment]
@andrew-w-ross I'd be happy to help with any react typings.

**Pull request merged**